frontend/packages/console-app/src/components/quick-starts/controller/QuickStartIntroduction.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { SyncMarkdownView } from '@console/internal/components/markdown-view';
 import { QuickStartTask, QuickStartTaskStatus } from '../utils/quick-start-types';
 import TaskHeader from './QuickStartTaskHeader';
+import { pluralize } from '@patternfly/react-core';
```

Comment:
not a big thing, but should this import be ordered at top.

Reply:
Updated.